config/deploy.rb

```diff
@@ -19,7 +19,7 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-set :linked_files, %w{config/database.yml config/blacklight.yml config/honeybadger.yml config/secrets.yml config/schedule.rb public/robots.txt}
+set :linked_files, %w{config/database.yml config/blacklight.yml config/honeybadger.yml config/secrets.yml public/robots.txt}
```

Comment:
Note that puppet is managing the cronjobs in production, not whenever. So this change uses the git version of `config/schedule.rb` (which has the EDS cache cleanup)